packages/foreman/rubygem-fog-vsphere/rubygem-fog-vsphere.spec

```diff
@@ -88,6 +88,9 @@ cp -pa .%{gem_dir}/* \
 %{gem_instdir}/tests
 
 %changelog
+* Tue Nov 16 2018 Chris Roberts <chrobert@redhat.com> - 2.5.0-1
```

Comment:
This doesn't exist. Note that we have a `bump_rpm.sh` script which does the bumping for you.